src/pybind/mgr/dashboard/frontend/src/app/ceph/block/mirroring/daemon-list/daemon-list.component.html

```diff
@@ -1,13 +1,21 @@
-<cd-table [data]="data"
-          columnMode="flex"
-          [columns]="columns"
-          [autoReload]="-1"
-          (fetchData)="refresh()"
-          [status]="tableStatus">
-</cd-table>
+<div class="col-md-12">
+  <div class="daemon">
```

Comment:
Let's avoid custom classes
```suggestion
  <div>
```